docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -213,6 +228,85 @@ To add a Task to the end of a Mission:
 
     :::
 
+### Custom Tasks  {#custom-task}
+
+Users can create their own tasks following a specific template and place them within the **/onav_log/custom_tasks** directory.
+Restarting the UGV or dockers will trigger the system to load the custom task that was created, making it available for mission use.
+Below is a sample template file named "log_inputs.py" which is elaborated further in the following section.

```

Comment:
Is it actually logging inputs or publishing feedback? I didn't see any logging.

Reply:
I'll change it to "publish_inputs.py" and update the text accordingly to be more precise.